book/website/collaboration/inclusive-events.md

```diff
@@ -6,34 +6,37 @@
 It covers: 
 - planning and logistical considerations
 - guidance on how to evaluate and choose a location
-- considerations and requirement for developing a communications strategy
+- considerations and requirements for developing a communications strategy
 - suggestions for how to create an inclusive and accessible agenda
-- accessible technology requirements and reccomendations for hybrid events
+- accessible technology requirements and recommendations for hybrid events
 - and provides a checklist that can be tailored to your specific event
 
 
 (cl-inclusive-events-background)=
 ## Motivation and Background
 
-Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background. 
-This chapter is meant to serve as a resource to support and guide our community members with the tools to identify the opportunities and challenges in upholding this, and to approach event planning and design with accessibility and inclusion in mind.
+Every person should have the opportunity to participate fully and equally in all events and social activities – regardless of their individual ability, gender, age, ethnic or social background and there has been an increased effort in many disciplines to involve members of the public and people with lived experience in research projects. 
 
-Because both internal resources and the events you run can range in size - from from small meetings to massive multi-day conferences - this chapter is not an exhaustive list and instead is meant to provide general guidance about how to design events that are inclusive for people with lived experience. 
+This chapter is meant to serve as a resource to support and guide researchers, organizers, and community members with the tools to identify the challenges and opportunities in organizing events that involve and include people with lived experience.
+
+We refer to "people with lived experience" throughout this chapter as a way to recognize and acknowledge individuals who have direct personal knowledge, or firsthand experience of a particular situation, condition, or topic. It is a term often used to emphasise the importance of including the perspectives and insights of those who have had direct experience of a specific condition or situation. In the context of event planning, the considerations need to cover both the disability requirements of your attendees, and an identification of barriers that would be experienced by your specific attendees with lived experience. For example, attendees who are immunocompromised, experience long-term health conditions, people with disabilities, and those with caring responsibilities would all be considered as people with lived experience, and would have slightly different accommodations needed to support their engagement in an event. 
```

Comment:
> For example, attendees who are immunocompromised, experience long-term health conditions, people with disabilities, and those with caring responsibilities would all be considered as people with lived experience, and would have slightly different accommodations needed to support their engagement in an event. 

I think this is potentially a bit confusing. If the conference/event deals with these topics, then these attendees would have lived experience but otherwise that might not be the right label? E.g. at a conference on astrophysics, it would be weird to refer to someone with caring responsibilities as "a person with lived experience on the topic"? Just to be cautious to not conflate needs for accommodations with lived experience generally 🙂

Reply:
That makes sense. I can take out the "or topic" on line 38 currently - but I'm not sure how else to address it. 

I added topic because "carer in palliative care" is not necessarily a single experience - but more of a area/topic/field. 

I think the chapter's breadth in terms of needing to account for every case, every field/domain, internationalization, and reader base may mean that some statements have to be broad. 

Happy to take suggestions if you can think of a better phrasing.